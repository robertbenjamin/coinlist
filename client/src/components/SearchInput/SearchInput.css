.search-input {
  border: 1px solid #001f3f;
  width: 100%;
  padding: 10px;
  margin-bottom: 15px;
}

.search-input:focus {
  outline-width: 0;
}
