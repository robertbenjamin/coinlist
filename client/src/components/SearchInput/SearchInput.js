import React from 'react';
import './SearchInput.css';

const SearchInput = props => {
  return (
    <input
      type='text'
      placeholder='Search for a coin'
      value={props.searchText}
      onChange={props.handleSearchInput}
      className='search-input'></input>
  );
};

export default SearchInput;
