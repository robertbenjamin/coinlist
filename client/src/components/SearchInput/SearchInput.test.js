import React from 'react';
import SearchInput from '../SearchInput';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<SearchInput />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
