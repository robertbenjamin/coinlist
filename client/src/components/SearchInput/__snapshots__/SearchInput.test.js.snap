// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<input
  className="search-input"
  placeholder="Search for a coin"
  type="text"
/>
`;
